packages/syntax-core/src/getZIndex.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * [getZIndex](?path=/docs/ZIndex--docs) is a utility function that returns a z-index value based on the layer and local layer.
+ *
+ * Usage:
+ * ```
+ * <Navbar zIndex={getZIndex("sticky")} />
+ * <Modal zIndex={getZIndex("modal")} />
+ * ```
+ */
+export default function getZIndex(
```

Comment:
How are you thinking the migration process would work, to move from our current z-indexes to these? Would we need to migrate all z-indexes at once?